src/test/bridges/liquity/TroveBridgeUnit.t.sol

```diff
@@ -63,7 +64,7 @@ contract TroveBridgeTest is TestUtil {
         vm.label(address(bridge.LUSD_USDC_POOL()), "LUSD_USDC_POOL");
         vm.label(address(bridge.USDC_ETH_POOL()), "USDC_ETH_POOL");
         vm.label(address(LIQUITY_PRICE_FEED), "LIQUITY_PRICE_FEED");
-        vm.label(0x66017D22b0f8556afDd19FC67041899Eb65a21bb, "STABILITY_POOL");
+        vm.label(STABILITY_POOL, "STABILITY_POOL");
         vm.label(0x4f9Fbb3f1E99B56e0Fe2892e623Ed36A76Fc605d, "LQTY_STAKING_CONTRACT");
```

Comment:
Why is the `LQTY_STAKING_CONTRACT` not using a constant similar to the stability pool?

Reply:
Addressed in 1b91dfd8ff94a82d751e73fa54e23673f2d79ab4